src/mixed-reality/HISTORY.rst

```diff
@@ -0,0 +1,13 @@
+.. :changelog:
+
+Release History
+===============
+
+0.0.3
+++++++
+Add new command `az remote-rendering-account`
+Add new command for `az spatial-anchors-account`
+
+0.0.1
+++++++
+* Initial release.
```

Comment:
Just out of curiosity, why is there no history of version 0.0.2 here? If you can find the history, please add it~

Reply:
Unfortunately, it seems that the first author forgot to write history for '0.0.2'.